common-custom-user-data-gradle-plugin/settings.gradle

```diff
@@ -4,3 +4,11 @@ plugins {
 }
 
 rootProject.name = 'common-custom-user-data-gradle-plugin'
+
+gradleEnterprise {
+    buildScan {
+        termsOfServiceUrl = "https://gradle.com/terms-of-service"
+        termsOfServiceAgree = "yes"
+        publishAlways()
+    }
+}
```

Comment:
We cannot check this in @runningcode . People need to have a choice to agree or not. They can simply run the build with `--scan`.

Reply:
Ah sorry my goal here was to try to trigger the CI pipeline by creating a PR. I've updated the description to make that clear.